src/components/Trade/hooks/useTradeRoutes/useTradeRoutes.test.tsx

```diff
@@ -23,6 +23,12 @@ jest.mock('state/slices/assetsSlice/assetsSlice', () => ({
   })
 }))
 
+jest.mock('@shapeshiftoss/market-service', () => ({
+  findAll: jest.fn,
+  findByCaip19: jest.fn,
+  findPriceHistoryByCaip19: jest.fn
+}))
+
```

Comment:
we should be able to create a common mock in `__mocks__`

Reply:
@cjthompson I have tried creating __mocks__/@shapeshiftoss/market-service.ts file but was unable to make it work with jest whatever I tried. Please suggest if you have any resource or example I could use. Only thing I was able to do was to load it directly in setupTests.ts, which is not satisfactory.

Reply:
@chxash it would be nice if we could come up with some other approach, as it's duplicated verbatim 9 times and modified once.

Reply:
@cjthompson @0xdef1cafe created new file setupGlobalMock.ts to have this mocking in a single place